KeenClient.podspec

```diff
@@ -21,8 +21,8 @@ Pod::Spec.new do |spec|
   spec.source       = { :git => 'https://github.com/keenlabs/KeenClient-iOS.git', :tag => spec.version.to_s }
   spec.source_files = 'KeenClient/*.{h,m}','Library/Reachability/*.{h,m}'
   spec.public_header_files = 'KeenClient/*.h'
-  spec.frameworks   = 'CoreLocation'
   spec.requires_arc = true
+  spec.frameworks = 'SystemConfiguration', 'CoreLocation', 'CFNetwork'
```

Comment:
In PR #176 it was mentioned that maybe it would be possible/useful to make CFNetwork optional. Is that a thing? Is it important? Is it possible or even desirable to do that? What are the ramifications of this dependency? If it's a big deal, should there be a blurb in the README about dragging it in? Just wanted to understand that comment in the other PR.